app.rb

```diff
@@ -0,0 +1,108 @@
+# rubocop:disable Metrics/MethodLength
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './rental'
+# Decare a variables to hold lists
+@books = []
+@persons = []
+@rentals = []
+@classroom = Classroom.new('CR 1001')
```

Comment:
[REQUIRED] I can't review your code because it is returning an error. Kindly fix this. and resubmit.

|Your App|
|--------|
|<img width="1370" alt="Screenshot 2022-10-01 at 10 36 57" src="https://user-images.githubusercontent.com/62938735/193403278-70aafe79-adcc-43a0-8694-6ca8fd7f471c.png">|

Reply:
@fobadara Thanks for your review! I have made some changes and now I am stuck with creating the person. I need little assistance on that. thanks again!
![Screenshot from 2022-10-01 14-33-30](https://user-images.githubusercontent.com/75738563/193407763-fda32797-9cb6-4f02-b41d-792e8c42c8a8.png)

Reply:
I think this might be the problem:

![image](https://user-images.githubusercontent.com/62938735/193408453-59245535-4004-41c1-b928-a8ccc4d41673.png)


![image](https://user-images.githubusercontent.com/62938735/193408488-97e9eaee-9632-4a6c-b196-e6920d345e81.png)

Reply:
Oh, I will check that. Thanks Abdu!

Reply:
You are welcome. And I prefer Abdulfatai or Fobadara